modules/http/src/test/java/org/glassfish/grizzly/http/HttpRequestParseTest.java

```diff
@@ -140,6 +141,19 @@ public void testSimpleHeadersPreserveCase() throws Exception {
         doHttpRequestTest("POST", "/index.html", "HTTP/1.1", headers, "\r\n", true);
     }
 
+    @Test
+    public void testLeadingOrTrailingWhitespaceFromHeaderContentValues() throws Exception {
+        Map<String, Pair<String, String>> headers = new HashMap<>();
+        headers.put("Host", new Pair<>("localhost", "localhost"));
+        headers.put("Leading-OWS-Header1", new Pair<>("  some-value", "some-value"));
+        headers.put("Leading-OWS-Header2", new Pair<>("\t\tsome-value", "some-value"));
+        headers.put("Trailing-OWS-Header1", new Pair<>("some-value  ", "some-value"));
+        headers.put("Trailing-OWS-Header2", new Pair<>("some-value\t\t", "some-value"));
+        headers.put("OWS-Header", new Pair<>(" \t \t some-value \t \t ", "some-value"));
+        headers.put("Mixed-OWS-Header", new Pair<>(" \t \t some- \t \t value \t \t ", "some- \t \t value"));
+        doHttpRequestTest("POST", "/index.html", "HTTP/1.1", headers, "\r\n");
```

Comment:
Unclear for me.

I see it's tested that headers with `WS`es are still parsable.
But is there a test to verify that values are actually stripped of `WS`es?